viewer_clients/cmdline_client/cmd_line_client.py

```diff
@@ -41,63 +44,74 @@ def default(self, obj):
         d.update(obj.__dict__)
         return d
 
+
 def handle_auth_requests(args):
     session = session_manager.SessionManager_Client()
+
+    auth_token = session.getToken(args.host, args.port)
+
     auth_client = authentication_helper.ThriftAuthHelper(args.host,
                                                          args.port,
                                                          '/Authentication',
-                                                         session.getToken(
-                                                             args.host,
-                                                             args.port))
-
+                                                         auth_token)
     try:
         handshake = auth_client.getAuthParameters()
+
+        if not handshake.requiresAuthentication:
+            print("This server does not require privileged access.")
```

Comment:
The idea here is that this was started as a "separate tool" so it should work without most of the modules in the codechecker_lib modules that is why not the logger was used. To fix the verbosity levels the logger module needed to be imported, we might change the logging here too.
Printing the tables for results/runs should be done using the print not the LOG because of the formatting.